app/src/routes/browse.js

```diff
@@ -3,10 +3,16 @@ import React, { useState, useEffect } from "react";
 import { firebaseDb } from "../firebase/firebase";
 import { collection, onSnapshot } from "firebase/firestore";
 import Selector from "../components/selector";
+import MultiStepForm from "../components/multistepform";
+import MultiStepFormPage from "../components/multistepformpage";
 
 export default function Browse() {
   const [selectedJob, setSelectedJob] = useState(null);
   const [jobs, setJobs] = useState([]);
+  const [filters, setFilters] = useState({
+    "materials": ["PLA", "ABS", "PETG"],
+    "bid_order": 0,
+  });
 
   useEffect(() => {
     const unsubscribe = onSnapshot(
```

Comment:
Not sure what happened here but jobs no longer render on the board. Even after creating a new issue and refreshing the page, nothing shows up